<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateChat</class>
 <widget class="QWidget" name="CreateChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>129</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>248</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="chatTitleLabel">
       <property name="text">
        <string>Chat Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="chatTitleLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="chatThemeLabel">
       <property name="text">
        <string>Chat Theme</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="chatThemeComboBox">
       <item>
        <property name="text">
         <string>Froth</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Deep Sea</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ocean</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Seaweed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Beach</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Riverbed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jelly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Coral</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>248</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>129</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
